setup.cfg

```diff
@@ -26,7 +26,7 @@ known_first_party = rest_framework_guardian,tests
 
 [metadata]
 name = djangorestframework-guardian
-version = 0.2.0
+version = 0.3.0
```

Comment:
Please change back the version number. This is updated during the release process.